docs/content/en/install/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Install K8ssandra"
-linkTitle: "Install"
+title: "Install K8ssandra Operator"
```

Comment:
My comments from the other page about what we want to call the thing we're installing will apply over here as well.

Reply:
Just did a commit to fix this.